convert.sh

```diff
@@ -0,0 +1,139 @@
+#!/bin/env bash
+cd "$(dirname ${BASH_SOURCE[0]})"
+
+printf "%s\n" \
+    "-=- Luxor to OpenSMCE Game Converter -=-" \
+    "" \
+    "===========================================" \
+    "Step 0. Checking the requirements..." \
+    "==========================================="
+good=1
+
+python --version > /dev/null
+if [ $? -ne 0 ]; then
+    printf "! Python is not installed !\n"
+    good=0
+fi
+
+if [ ! -d "./data" ]; then
+    printf "! Folder "data" does not exist !\n"
```

Comment:
Looks like the quotation marks here are left unescaped, the script may crash if the execution gets there.